llvm/lib/Frontend/OpenMP/OMPIRBuilder.cpp

```diff
@@ -7053,13 +7053,28 @@ OpenMPIRBuilder::InsertPointTy OpenMPIRBuilder::emitTargetTask(
                     << "\n");
   return Builder.saveIP();
 }
+
 static void emitTargetCall(
     OpenMPIRBuilder &OMPBuilder, IRBuilderBase &Builder,
     OpenMPIRBuilder::InsertPointTy AllocaIP, Function *OutlinedFn,
     Constant *OutlinedFnID, int32_t NumTeams, int32_t NumThreads,
     SmallVectorImpl<Value *> &Args,
     OpenMPIRBuilder::GenMapInfoCallbackTy GenMapInfoCB,
     SmallVector<llvm::OpenMPIRBuilder::DependData> Dependencies = {}) {
+  //  emitKernelLaunch
```

Comment:
Minor nit : Consider making it clear in the comment here that this is a host-to-host launch?

Reply:
Done.